core/web/bridge_types_controller.go

```diff
@@ -154,6 +154,15 @@ func (btc *BridgeTypesController) Destroy(c *gin.Context) {
 		jsonAPIError(c, http.StatusConflict, fmt.Errorf("can't remove the bridge because jobs %v are associated with it", jobsUsingBridge))
 		return
 	}
+	v2jobsUsingBridge, err := btc.App.GetJobORM().FindJobIDsWithBridge(name)
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, fmt.Errorf("error searching for associated jobs for BTC Destroy: %+v", err))
```

Comment:
BTC?

Reply:
Bridge Type Controller I guess - was used in some of the older error strings. Removed it